pkg/interactive-installer/Dockerfile

```
@@ -0,0 +1,53 @@
+# syntax=docker/dockerfile-upstream:1.5.0-rc2-labs
+FROM lfedge/eve-alpine:1f7685f95a475c6bbe682f0b976f12180b6c8726 AS build
+SHELL ["/bin/ash", "-eo", "pipefail", "-c"]
+
+ENV BUILD_PKGS curl build-base git ncurses-dev musl-dev python3 bash
+ENV PKGS ncurses-dev smartmontools mtools strace util-linux e2fsprogs e2fsprogs-extra jq
+
+RUN eve-alpine-deploy.sh
+ENV RUSTUP_HOME=/usr/local
+ENV CARGO_HOME=/usr/local/cargo
+ENV CARGO_TARGET_DIR=/usr/local/my-installer/target
+ENV RUST_VERSION=1.79.0
+ADD https://static.rust-lang.org/dist/rust-${RUST_VERSION}-x86_64-unknown-linux-musl.tar.gz rust-${RUST_VERSION}-x86_64-unknown-linux-musl.tar.gz
+ADD https://static.rust-lang.org/dist/rust-${RUST_VERSION}-aarch64-unknown-linux-musl.tar.gz rust-${RUST_VERSION}-aarch64-unknown-linux-musl.tar.gz
+
+# Install Rust using the standalone installer
+RUN ARCH="aarch64" && \
+    if [ "$(uname -m)" == "x86_64" ]; then \
+        ARCH="x86_64"; \
+    fi && \
+    tar -xzf rust-${RUST_VERSION}-${ARCH}-unknown-linux-musl.tar.gz && \
+    cd rust-${RUST_VERSION}-${ARCH}-unknown-linux-musl && \
+    ./install.sh && \
+    cd .. && \
+    rm -rf rust-${RUST_VERSION}-${ARCH}-unknown-linux-musl rust-${RUST_VERSION}-${ARCH}-unknown-linux-musl.tar.gz
+
+ENV PATH "$PATH:/usr/local/cargo/bin"
+
+WORKDIR /usr/local/my-installer
+COPY src src
+COPY Cargo.lock Cargo.lock
+COPY Cargo.toml Cargo.toml
+COPY rust-toolchain.toml rust-toolchain.toml
+
+# Add the vendor tar file
+COPY vendor_part_* /usr/local/my-installer/
+RUN cat /usr/local/my-installer/vendor_part_* > vendor.tar.gz \
+    && tar xf vendor.tar.gz \
+    && rm vendor.tar.gz vendor_part_*
```

Isn't it possible to just use a pipe?
```suggestion
RUN cat /usr/local/my-installer/vendor_part_* | tar xf - 
```